### PR TITLE
ovirt: PEP8 compliancy fixes

### PR DESCRIPTION
**Pull request by sbulage**
SUMMARY

This PR includes:

PEP8 compliancy fixes

ISSUE TYPE

Feature Pull Request
Docs Pull Request
COMPONENT NAME

ovirt/ovirt*
misc/ovirt

ANSIBLE VERSION

v2.5

### CODE REVIEW

**Comment by Akasurde** [comment]
@sbulage Could you please rebase ?

**Comment by sbulage** [comment]
@Akasurde This branch/code is rebased. :)

**Review by mwperina — APPROVED** [review]
Looks good to me, thanks

**Comment by resmo** [comment]
shipit

**Review by resmo — APPROVED** [review]

File: test/sanity/pep8/legacy-files.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@ lib/ansible/modules/cloud/amazon/_ec2_ami_search.py
 lib/ansible/modules/cloud/amazon/_ec2_remote_facts.py
 lib/ansible/modules/cloud/amazon/_ec2_vpc.py
 lib/ansible/modules/cloud/amazon/_ec2_vpc_dhcp_options.py
+lib/ansible/modules/cloud/openstack/_os_server_actions.py
```
Comment (Akasurde) [inline test/sanity/pep8/legacy-files.txt]: This file passes current set of rules, so no need to add it again.

```
# python2.7 /usr/local/bin/pycodestyle --max-line-length 160 --config /dev/null --ignore E305,E402,E722,E741 lib/ansible/modules/cloud/openstack/_os_server_actions.py && echo $?
0
```

**Review by Akasurde — CHANGES_REQUESTED** [review]
Rest LGTM

**Pull request merged**